pkg/users/account-create-dialog.js

```diff
@@ -118,6 +130,23 @@ function validate_real_name(real_name) {
         return _("The full name must not contain colons.");
 }
 
+function validate_uid(uid, accounts, change) {
+    if (!uid)
+        return _("No UID specified");
+
+    const uid_number = Number(uid);
+    if (!Number.isInteger(uid_number) || uid_number < 0)
+        return _("UID must be a positive integer.");
+
+    if (uid_number < 1000)
+        return _("UID below 1000 is reserved for system users.");
```

Comment:
This shouldn't be hardcoded, but use the values in `/etc/login.defs`, such as `UID_MIN` (`1000` on my Fedora system) and perhaps `UID_MAX` (`60000` on my system).

BTW: If we're already hardocding this elsewhere (for users or groups), we should change it there too.

It's probably 1000 on most distributions by convention, but it isn't necessarily so.

Reply:
For groups we already read `/etc/login.defs` so this can be made re-usable:

```
pkg/users/group-create-dialog.js:    cockpit.file("/etc/login.defs").read()
```

Reply:
Fixed